docs/examples/containerized_heketi_dedicated_gluster/README.md

```diff
@@ -0,0 +1,669 @@
+the container picks up the actual ipaddr of the node it is running on.
+
+#### Test connection to Heketi.
+
+```
+
+	# curl http://k8dev3.rhs:8888/hello
+	Hello from Heketi
+```
+
+#### Set an environment variable for the Heketi Server.
+In a previous step we have installed the Heketi-Client on our 
+master node.
+
+
+```
+	# export HEKETI_CLI_SERVER=http://k8dev3.rhs:8888
+```
+
+### Using Heketi with Gluster
+
+Topology is used to tell Heketi about the environment, what nodes and devices it will manage.
+
+***NOTE:*** Heketi is currently limited to managing raw devices only, if a device is already a Gluster volume it
+will be skipped and ignored.
+
+
+
+#### Create and load the topology file.
+
+There is a sample file located in /etc/heketi/heketi-client/share/heketi/topology-sample.json depending on where you unpacked the client from previous step.
+The lsblk command run from the GlusterFS nodes can help determine what devices are available, for this example I will only load the /dev/sdc device into the
+topology.
+
+```
+	# lsblk
+	NAME                                                                              MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+	sda                                                                                 8:0    0  100G  0 disk 
+	└─vg_brick1-lv_brick1                                                             253:2    0   20G  0 lvm  /mnt/registry
+	sdb                                                                                 8:16   0  100G  0 disk 
+	└─vg_brick2-lv_brick2                                                             253:23   0   80G  0 lvm  /mnt/heketimgmt
+	sdc                                                                                 8:32   0  100G  0 disk <1>
+	sdd                                                                                 8:48   0  100G  0 disk <2>
+	sde                                                                                 8:64   0  100G  0 disk <3>
+```
+<1> Available device as /dev/sdc
+
+<2> Available device as /dev/sdd
+
+<3> Available device as /dev/sde
+
+
+Example topology.json file
+```
+{
+  "clusters": [
+    {
+      "nodes": [
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster25.rhs"
+              ],
+              "storage": [
+                "192.168.1.205"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sdc"
+          ]
+        },
+        {
+          "node": {
+            "hostnames": {
+              "manage": [
+                "gluster26.rhs"
+              ],
+              "storage": [
+                "192.168.1.206"
+              ]
+            },
+            "zone": 1
+          },
+          "devices": [
+            "/dev/sdc"
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
```

Comment:
Include a note about how "manage" should be hostnames and "storage" should be IP addresses. I can guarantee you someone will stumble over this by not having read it in the three-ish other places we have it noted. :)